### ISSUE TITLE
Option to remove $format parameter

### ISSUE THREAD

**Issue by pkunze** [janhommes/o.js#39]
Is there a way to omit the `$format=json` part from the generated query string? I would love to use this library for my extension libraries for MS Dynamics CRM but all the requests fail with the following resonse:

```xml
<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<error xmlns="http://schemas.microsoft.com/ado/2007/08/dataservices/metadata">
  <code></code>
  <message xml:lang="de-DE">The query parameter '$format' begins with a system-reserved '$' character but is not recognized.</message>
</error>
```

[OData V4 Url convention](http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part2-url-conventions.html) (section 5.1) doesn't imply that this query option in mandatory anyways (just like any other queryoption, really) so maybe this should just be present when the `format`-option is beeing set?

**Comment by pkunze** [comment]
I this package activeley maintained? If so, id like to submit a PR with a minor change so solve the issue.  
I also found out that the `json`-config option isn't beeing used anymore..

**Comment by janhommes** [comment]
Hi pkunze,

Basically this repo is not actively maintained. The main reason is, that odata.org changed their example odata service and all tests are now failing. That is also the reason, I did not accept the last PR which already fixed some issues, because it will need rework on the test. I tried to find a working odata-sample server which fits our needs (cross-origin, no auth), but could not find any in time.

When somebody is willing to check the tests or set up a new test-server, this repo maybe will life on. I am sorry, I have not much to do with odata any more and moved on to other techs and do not have the time to set up one by my own. Otherwise  I need to mark this repo as not maintained.

Regards
Jan
